modules/lvms-limitations-to-configure-size-of-devices.adoc

```diff
@@ -23,36 +23,54 @@ ifdef::microshift[]
 endif::microshift[]
 
 ifndef::microshift[]
-.Size limits for different architectures using the default PE and LE size
-[cols="1,1,1,1,1", width="100%", options="header"]
+.Tested configuration
+[cols="1,1", width="100%", options="header"]
 |====
-|Architecture
-|RHEL 6
-|RHEL 7
-|RHEL 8
-|RHEL 9
 
-|32-bit
-|16 TB
-|-
-|-
-|-
+|Parameter
+|Value
 
-|64-bit
+|Chunk size
```

Comment:
Unit style matches the IBM SG:
![Screenshot from 2024-07-29 12-04-36](https://github.com/user-attachments/assets/cc8cce44-662e-47f8-9a28-8009b7a02177)